frontend/src/scenes/settings/project/SessionRecordingSettings.tsx

```diff
@@ -177,6 +178,134 @@ export function ReplayAuthorizedDomains(): JSX.Element {
+                        updateCurrentTeam({
+                            session_recording_summary_config: {
+                                ...currentConfig,
+                                opt_in: checked,
+                            },
+                        })
+                    }}
+                    label="Opt in to enable AI suggested summaries"
+                />
+            </div>
+            <div>
+                <h3 className="flex items-center gap-2">
+                    <IconSelectEvents className="text-lg" />
+                    Preferred events
+                </h3>
+                <p>
+                    These events are treated as more interesting when generating a summary. We recommend you include
+                    events that represent value for your user
+                </p>
+                <EventSelect
+                    onChange={(includedEvents) => {
+                        updateCurrentTeam({
+                            session_recording_summary_config: {
+                                ...currentConfig,
+                                preferred_events: includedEvents,
+                            },
+                        })
+                    }}
+                    selectedEvents={currentConfig.preferred_events || []}
+                    addElement={
+                        <LemonButton size="small" type="secondary" icon={<IconPlus />} sideIcon={null}>
+                            Add event
+                        </LemonButton>
+                    }
+                />
+            </div>
+            <div>
+                <h3 className="flex items-center gap-2">
+                    <IconSelectEvents className="text-lg" />
+                    Excluded events
+                </h3>
+                <p>These events are never submitted even when they are present in the session.</p>
+                <EventSelect
+                    onChange={(excludedEvents) => {
+                        updateCurrentTeam({
+                            session_recording_summary_config: {
+                                ...currentConfig,
+                                excluded_events: excludedEvents,
+                            },
+                        })
+                    }}
+                    selectedEvents={currentConfig.excluded_events || []}
+                    addElement={
+                        <LemonButton size="small" type="secondary" icon={<IconPlus />} sideIcon={null}>
+                            Exclude event
+                        </LemonButton>
+                    }
+                />
+            </div>
+            <div>
+                <h3 className="flex items-center gap-2">
+                    <IconSelectEvents className="text-lg" />
+                    Included event properties
+                </h3>
+                <p>
+                    We always send the event name and timestamp. The only other data sent are values of the properties
```

Comment:
Certainly fine for v1 but we might get customers looking for per event property selection